include/swift/AST/Module.h

```diff
@@ -1290,6 +1289,7 @@ class ModuleEntity {
   bool isBuiltinModule() const;
   const ModuleDecl *getAsSwiftModule() const;
   const clang::Module *getAsClangModule() const;
+  void *getOpaqueValue() const;
```

Comment:
I'd recommend to make this an inline method (move the definition inline). The definition is simple and inlining it would be good for performance.